app/views/examples/header-and-footer/index.njk

```diff
@@ -0,0 +1,48 @@
+{% extends "full-width.njk" %}
+
+{% from "skip-link/macro.njk" import govukSkipLink %}
+{% from "header/macro.njk" import govukHeader %}
+{% from "phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "footer/macro.njk" import govukFooter %}
+
+{% block content %}
+
+{# {{ govukSkipLink({
+  href: '#content',
+  text: 'Skip to main content'
+}) }} #}
```

Comment:
Why is the code commented out? Is it opt in by uncommenting it? Could the code mention this?

Reply:
Wasn't sure if the example should contain the skip-link or not. I added it back now. I think is useful when testing. Would be good to have the cookie-banner added to when it will be in.

Regarding the other comments: I tested with the ATs already. Mobile menu may be expanded by default, but that means there will be movement in the page when javascript kicks in and hides it.